Tests/Data/Elliptic/square_1x1_GroundWaterFlow/square_1e6_with_nodal_sources.prj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 <OpenGeoSysProject>
     <mesh>square_1x1_quad_1e6.vtu</mesh>
-    <geometry>square_1x1.gml</geometry>
```

Comment:
Only curious: Why is the midpoint necessary?

Reply:
The midpoint is used for the nodal source term in that ctest.